sphinxserve/__init__.py

```diff
@@ -48,6 +52,13 @@
                         short: d
                         action: store_true
                         default: __SUPPRESS__
```

Comment:
You are right! Yes, I totally forgot (I've been working on the dev branch for a while) that is the default in master and seems your usecase will benefit from more states. We can use loglevel 60 for quiet mode and deprecate debug. Lets go ahead an introduce those changes. (in the next iteration we can translate loglevels 0-5 to match 60-10 with 0 for quiet and 5 for maximum debug, as it seems much more natural for everybody). Until release 1.0 we are not that concern about backward compatibility, but that we have a clean API.

BTW, the latest code I am using is already in dev which brings back full python3 compatibility, some code refactor and less depencencies. The plan is to release it for everybody on the Pycon sprints. It would be great if you can test it too.